index.css

```diff
@@ -8,10 +18,178 @@
   letter-spacing: 4px;
   text-align: center;
   transition: transform 1s;
+  position: relative;
   -webkit-background-clip: text;
 }
 
 .logo:hover {
   transform: scale(1.2);
   -webkit-text-stroke: 0.2px rgba(10, 78, 10, 0.664);
+}
+
+.header {
+  align-items: center;
+  background-color: rgba(216, 143, 143, 0.5);
+  display: flex;
+  flex-direction: column;
+  height: 200px;
+  justify-content: center;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  width: 100vw;
+}
+
+.links {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 10px;
+}
+
+.nav-links::after {
+  content: "";
+  background-color: rgba(34, 49, 12, 0.404);
+  display: flex;
+  flex-direction: row;
```

Comment:
These properties are no needed. It makes no difference to have them in the code or not.